journal/modernized/08-chapter-5.adoc

```diff
@@ -151,8 +151,8 @@ and to do whatsoever he is pleased to call for.
 
 Some time after this, as I was returning home late one evening,
 I had a sense of invitation after this manner,
-'What if the Emperor of Russia should need a person for the superintendence
-of agriculture--at which time a willingness was begotten in my mind to go,
+"What if the Emperor of Russia should need a person for the superintendence
+of agriculture?"--at which time a willingness was begotten in my mind to go,
```

Comment:
These ones also need the backtick ` inside of the quotations marks, (as I saw you were chatting with Jared about)